src/server/config.js

```diff
@@ -14,3 +20,10 @@ module.exports = {
   version: require('../../package').version,
   webpackStylesExtensions: ['css', 'less', 'sass', 'scss', 'styl']
 };
+
+// Use above config as a default one
+// Multiple other providers are available like loading config from json and more
+// Check out nconf docs for fancier examples
+nconf.defaults(config);
+
+module.exports = nconf.get();
```

Comment:
Ah, sorry! Will plug it in and test all the further PR's that I need to tweak today anyway.